### PR TITLE
KJHH-2137: Organisation select: resolve parent path

### PR DESCRIPTION
**Pull request by jollem**

### CODE REVIEW

File: src/main/static/src/actions/organisaatio.actions.ts [hunk 1]
```diff
@@ -157,3 +158,15 @@ export const fetchOrganisations = (oidOrganisations: Array<string>) => (dispatch
...
+export const fetchOrganisationNames = () => async (dispatch: Dispatch) => {
+    const url = urls.url('kayttooikeus-service.organisaatio.names');
+    http.get<OrganisaatioNameLookup>(url)
+        .then((payload) =>
+            dispatch({
+                type: FETCH_ORGANISATION_NAMES,
+                payload,
+            })
+        )
+        .catch((error) => console.log('Nimien lataus epäonnistui', error));
```
Comment (mkamka) [inline src/main/static/src/actions/organisaatio.actions.ts:171]: Is this logging necessary and could it be implemented into some error notification if it is avail?

File: src/main/static/src/components/common/select/OrganisaatioSelect.tsx [hunk 2]
```diff
@@ -92,10 +101,14 @@ export class OrganisaatioSelect extends React.Component<OrganisaatioSelectProps,
             : null;
     };
 
+    _resolveName = (oid: string, locale: string): string => this.props.organisationNames?.[oid]?.[locale] || oid;
+
     _renderParents = (organisaatio: OrganisaatioSelectObject): React.ReactNode => {
-        return organisaatio.parentNames.map((parent: string, index: number) => (
-            <span key={index} className="parent">
-                {parent} &gt;{' '}
+        const path = organisaatio.oidPath.split('/');
```
Comment (mkamka) [inline src/main/static/src/components/common/select/OrganisaatioSelect.tsx:107]: Why has this changed? I think a better place might be somewhere in https://github.com/Opetushallitus/henkilo-ui/blob/08b7440b67ebc6fcb262ee6d3cbb4081cc2bd138/src/main/static/src/utilities/organisaatio.util.tsx#L53

File: src/main/static/src/actions/organisaatio.actions.ts (hunk 1 above)
Comment (jollem) [reply src/main/static/src/actions/organisaatio.actions.ts:171]: It is not fatal is fetch fails, implementation shows oid:s instead. console log is there just to give a hint that something went wrong.
currently notification mechanism is such that building new notifications is quite laborsome and in this case pointless.

File: src/main/static/src/components/common/select/OrganisaatioSelect.tsx (hunk 2 above)
Comment (jollem) [reply src/main/static/src/components/common/select/OrganisaatioSelect.tsx:107]: to keep impact minimal. currently the only known usage is within this component. given the non-existent test coverage, didn't want to touch functions manipulating common datastructures

**Review by mkamka — APPROVED** [review]
Täähän on ihan timanttia! Hyvää duunia!!!

**Review by mikaelbarlund — APPROVED** [review]

**Pull request merged**